src/bugsnag/ErrorPrompt.scss

```diff
@@ -0,0 +1,121 @@
+// NOTE: do not include any other files here, keep these styles separate from the rest
+
+.ErrorPrompt {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+
+  background: rgba(0, 0, 0, 0.6);
```

Comment:
>Given this is imported in the main index.js it'll be bundled into all of the other dependencies that are included there as well anyway, so this may have little 'real world' impact/benefit.


I agree with this comment, I don't think there would be a huge difference if you import `constants.scss` and use them in this file. Can you give us some examples for the potential errors we are avoiding by encapsulating these styles from the rest?

Reply:
If you have strong opinion about not using the global scss file at least add these values as constants at the top of this file.

Reply:
The rational is simple: https://github.com/sparkletown/sparkle/pull/1858#discussion_r674188044

Added to it, I've opted to use the simplest UI, no "fancy" hover effects or anything that depends on outside styles (as is `constants.scss`), as it is not about performance, but a principle of isolation/compartmentalization.

Given that principle, "constants.scss is being included by the entire rest of the application" is definitely adding a lot of reasons for some one to go in `constants.scss` and change stuff potentially introducing unwanted style changes, least bugs.

Reply:
I've gone through the SCSS file and couldn't find same color being used in 2 places warranting a variable.

 NOTE: colors (and styles in general) being used are from https://www.figma.com/file/WvrLkcfGAbyUvRl29AwvdQ/Components?node-id=233%3A0&viewport=132%2C210%2C0.4309226870536804